browser-switch/src/test/java/com/braintreepayments/browserswitch/BrowserSwitchClientTest.java

```diff
@@ -304,41 +514,40 @@ public void deliverResult_whenRequestIsPending_clearsResultStoreAndNotifiesResul
         assertNotNull(result);
         assertEquals(result.getStatus(), BrowserSwitchResult.STATUS_CANCELED);
         assertNull(result.getErrorMessage());
+        assertSame(result.getRequestMetadata(), requestMetadata);
 
         verify(persistentStore).clearActiveRequest(applicationContext);
     }
 
     @Test
     public void deliverResult_whenRequestIsNull_doesNothing() {
-        when(fragmentListener.getActivity()).thenReturn(activity);
-        when(activity.getApplicationContext()).thenReturn(applicationContext);
+        when(plainActivity.getApplicationContext()).thenReturn(applicationContext);
 
         when(persistentStore.getActiveRequest(applicationContext)).thenReturn(null);
         sut = BrowserSwitchClient.newInstance(browserSwitchConfig, activityFinder, persistentStore, returnUrlScheme);
-        sut.deliverResult(fragmentListener, browserSwitchListener);
+        sut.deliverResult(plainActivity, browserSwitchListener);
 
         verify(browserSwitchListener, never()).onBrowserSwitchResult(anyInt(), any(), any());
-        verify(persistentStore, never()).clearActiveRequest(activity);
+        verify(persistentStore, never()).clearActiveRequest(plainActivity);
     }
 
     @Test
     public void deliverResult_whenFragmentIsNotABrowserSwitchListener_throwsIllegalArgumentException() {
         exceptionRule.expect(IllegalArgumentException.class);
         exceptionRule.expectMessage("Fragment must implement BrowserSwitchListener.");
 
-        Fragment fragment = mock(Fragment.class);
         sut = BrowserSwitchClient.newInstance(browserSwitchConfig, activityFinder, persistentStore, returnUrlScheme);
-        sut.deliverResult(fragment);
+        sut.deliverResult(plainFragment);
```

Comment:
What is the meaning of the name `plainFragment`?

Reply:
Yeah it's an androidx `Fragment` that doesn't implement the `BrowserSwitchListener` interface.